src/app/page.tsx

```diff
@@ -1,5 +1,490 @@
+                  alt="Checkmark icon"
+                  width={32}
+                  height={32}
+                />
+                <p className="leading-[16px]">Retailer Markups</p>
+              </div>
+
+              <div className="flex flex-col gap-[8px]">
+                <Image
+                  src="/icons/buy-discount-shop-2-icon.svg"
+                  alt="Checkmark icon"
+                  width={32}
+                  height={32}
+                />
+                <p className="leading-[16px]">Overpackging</p>
+              </div>
+            </div>
+          </div>
+
+          <Button className="bg-blue text-white w-full font-bold h-[54px]" asChild>
+            <Link href="/about">Learn More About Supplement Club</Link>
+          </Button>
+        </div>
+      </div>
+
+      <div className="grid gap-y-[32px] lg:gap-y-[50px] pt-[48px] px-[16px] lg:p-[0]">
+        <div className="flex justify-between items-center">
+          <div className="grid gap-y-[16px]">
+            <p className="text-[32px] leading-[36px] font-[700] text-black">Products</p>
+            <p className="bg-white leading-[18px] text-grey1 py-[4px] px-[10px]">Next Drop: <span className="font-[700] ">January 1st 2025</span></p>
+          </div>
+
+          <Link href="/products" className="underline text-[18px] leading-[20px] text-black">View All</Link>
+        </div>
+
+        <div className="grid lg:grid-cols-3 gap-[16px]">
+          {products.map((product) => (
+            <ProductCardComponent key={product.id} {...product} />
+          ))}
+        </div>
+      </div>
+
+      <div className="w-full grid grid-cols-2 lg:grid-cols-7 mx-[16px] py-[80px] lg:p-[0]">
+
+        <div className="p-[12px] border-grey10 border-[1px] grid gap-[4px]">
+          <Image
+            src="/icons/heart-pulse-icon.svg"
+            alt="Checkmark icon"
+            width={40}
+            height={40}
+          />
+          <p className="text-[18px] leading-[24px] text-black">Heart Health</p>
+
+        </div>
+        <div className="p-[12px] border-grey10 border-[1px] grid gap-[4px]">
+          <Image
+            src="/icons/baby-icon.svg"
+            alt="Baby icon"
+            width={40}
+            height={40}
+          />
+          <p className="text-[18px] leading-[24px]  text-black">Fertility</p>
+
+        </div>
+        <div className="p-[12px] border-grey10 border-[1px] grid gap-[4px]">
+          <Image
+            src="/icons/bed-icon.svg"
+            alt="Bed icon"
+            width={40}
+            height={40}
+          />
+          <p className="text-[18px] leading-[24px] text-black">Sleep</p></div>
+
+        <div className="p-[12px] border-grey10 border-[1px] grid gap-[4px]">
+          <Image
+            src="/icons/energy-icon.svg"
+            alt="Energy icon"
+            width={40}
+            height={40}
+          />
+          <p className="text-[18px] leading-[24px] text-black">Energy</p></div>
```

Comment:
@VasylHulpak Please can we pass these into an array of objects and map through them instead, so we don't have to repeat the same code 14 times? 

See an example snippet below:

```
const data=[
    {
        label: 'Longevity',
       imageUrl: '/icons/dumbell-icon.svg' 
     },
     ......
]
```